src/erl_cache.erl

```diff
@@ -358,8 +358,10 @@ code_change(_OldVsn, State, _Extra) ->
     {ok, cache_opts()} | {error, invalid_opt_error()}.
 validate_opts(_, undefined) ->
     {error, {invalid, cache_name}};
+validate_opts([], Defaults) when Defaults =/= [] ->
+    {ok, Defaults};
```

Comment:
Can you add a quick inline comment here to note that this is only called with `Defaults == []` during initialization of options, and otherwise `Defaults` is passed as cached, validated, complete default set? Someday we'll come back and disambiguate _generating defaults_ from _validating input options to overlay on existing valid defaults_.